src/decorators/simpl.js

```diff
@@ -71,9 +74,14 @@ export function simpl(options) {
     } else {
       optionsWithDefaults.loadAllScenarios = false;
     }
+    if (options.hasOwnProperty('loadWorldResults')) {
+      optionsWithDefaults.loadWorldResults = options.loadWorldResults;
+    } else {
+      optionsWithDefaults.loadWorldResults = true;
```

Comment:
shouldn't we add the defaults values in `utils. wampOptionsWithDefaults()`?

Reply:
Why? const optionsWithDefaults = wampOptionsWithDefaults(options);

Reply:
This option is only used in the simpl decorator.

Reply:
good point